test/Microsoft.Health.Dicom.Tests.Integration/Features/CustomTagServiceTests.cs

```diff
@@ -0,0 +1,105 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Threading;
+using System.Threading.Tasks;
+using Dicom;
+using EnsureThat;
+using Microsoft.Health.Dicom.Core.Exceptions;
+using Microsoft.Health.Dicom.Core.Extensions;
+using Microsoft.Health.Dicom.Core.Features.CustomTag;
+using Microsoft.Health.Dicom.Core.Messages.CustomTag;
+using Microsoft.Health.Dicom.Tests.Common.Extensions;
+using Microsoft.Health.Dicom.Tests.Integration.Persistence;
+using Xunit;
+
+namespace Microsoft.Health.Dicom.Tests.Integration.Features
+{
+    public class CustomTagServiceTests : IClassFixture<CustomTagServiceTestsFixture>
+    {
+        private static readonly CancellationToken DefaultCancellationToken = new CancellationTokenSource().Token;
```

Comment:
Use the `default` keyword or `CancellationToken.None` throughout the tests since you're not using a `CancellationTokenSource`